dateutil/test/test_parser.py

```diff
@@ -180,7 +182,7 @@ def test_ybd(self):
             assert res == actual
 
 
-class TestInputFormats(object):
+class TestInputTypes(object):
```

Comment:
Why `Types` instead of `Formats`?

Reply:
Because whats being tested here is different types: bytes vs unicode vs bytearray vs StringIO...

Reply:
Ah, I was a bit thrown by the fact that the first test is just an empty string. I see now.